toolchain/mfc/params_tests/test_validate.py

```
@@ -0,0 +1,167 @@
+class TestValidateCase(unittest.TestCase):
+    """Tests for validate_case function."""
+
+    def test_valid_case_no_errors(self):
+        """Valid case should return empty errors."""
+        params = {
+            "m": 100,
+            "n": 50,
+            "p": 0,
+            "dt": 1e-5,
+        }
+        errors, warnings = validate_case(params)
```

Ruff (0.14.14)

[warning] Unpacked variable `warnings` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)